src/Disks/IDiskRemote.cpp

```diff
@@ -0,0 +1,491 @@
+#include <Disks/IDiskRemote.h>
+
+#if USE_AWS_S3
```

Comment:
Do we really need this if clause? I think DiskRemote can be independent of S3 library existence.